pkg/services/config.go

```diff
@@ -201,7 +194,11 @@ func New(ctx context.Context, config Config) (*Services, error) {
 	// that use postgres
 	config.DSN = strings.Replace(config.DSN, "postgresql://", "postgres://", 1)
 
-	credStore, credStoreEnv, err := credstores.Init(ctx, config.ToolRegistry, config.DSN, credstores.Options{
+	if len(config.ToolRegistries) < 1 {
+		config.ToolRegistries = []string{"github.com/obot-platform/tools"}
```

Comment:
`github.com/gptscript-ai/cmd`'s struct tag defaulting for `[]slice` types doesn't work. Setting it here until I can push a fix to that package.